napi-inl.h

```diff
@@ -1791,9 +1834,8 @@ inline ClassPropertyDescriptor<T> ObjectWrap<T>::InstanceMethod(
     InstanceMethodCallback method,
     napi_property_attributes attributes,
     void* data) {
-  CallbackData* callbackData = new CallbackData({}); // TODO: Delete when the class is destroyed
-  callbackData->instanceMethodCallback = method;
-  callbackData->data = data;
+  // TODO: Delete when the class is destroyed
+  InstanceMethodCallbackData* callbackData = new InstanceMethodCallbackData({ method, data });
```

Comment:
This is also from #18, so let’s discuss it there. Personally, I like to use it when it de-clutters the code at points where types are obvious anyway (e.g. I think it’s perfectly okay for the return value of `new Foobar()`), but prefer to avoid it when using `auto` would mean I have to look up the type.

I don’t think we have a fixed style guide; iirc even Nan and Node core don’t follow the same rules.